tests/integration/src/test/java/org/mule/test/construct/SubFlowTestCase.java

```diff
@@ -8,10 +8,10 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertThat;
+import static org.mule.functional.junit4.TestLegacyMessageUtils.getOutboundProperty;
 import static org.mule.test.core.lifecycle.LifecycleTrackerProcessor.FLOW_CONSRUCT_PROPERTY;
```

Comment:
this comment was just in case `LifecycleTrackerProcessor` would have been moved instead of copied, ignore.